2024-11-07 11:43:59,511 ERROR: Error fetching sheets: type object 'Sheet' has no attribute 'name' [in C:\Users\m\Desktop\web-based excel\custom-excel-app\backend\app.py:128]
2024-11-07 11:43:59,791 ERROR: Error fetching sheets: type object 'Sheet' has no attribute 'name' [in C:\Users\m\Desktop\web-based excel\custom-excel-app\backend\app.py:128]
2024-11-07 11:43:59,911 ERROR: Error fetching sheets: type object 'Sheet' has no attribute 'updated_at' [in C:\Users\m\Desktop\web-based excel\custom-excel-app\backend\app.py:128]
2024-11-07 11:44:08,270 ERROR: Error creating sheet: __init__() got an unexpected keyword argument 'name' [in C:\Users\m\Desktop\web-based excel\custom-excel-app\backend\app.py:163]
2024-11-07 13:09:28,763 ERROR: Error fetching sheets: type object 'Sheet' has no attribute 'updated_at' [in C:\Users\m\Desktop\web-based excel\custom-excel-app\backend\app.py:128]
2024-11-07 13:21:19,339 ERROR: Error fetching sheets: type object 'Sheet' has no attribute 'updated_at' [in C:\Users\m\Desktop\web-based excel\custom-excel-app\backend\app.py:128]
2024-11-07 13:21:19,348 ERROR: Error fetching sheets: type object 'Sheet' has no attribute 'updated_at' [in C:\Users\m\Desktop\web-based excel\custom-excel-app\backend\app.py:128]
2024-11-07 13:21:27,646 ERROR: Error creating sheet: __init__() got an unexpected keyword argument 'name' [in C:\Users\m\Desktop\web-based excel\custom-excel-app\backend\app.py:163]
2024-11-07 13:22:39,118 ERROR: Error fetching sheets: type object 'Sheet' has no attribute 'updated_at' [in C:\Users\m\Desktop\web-based excel\custom-excel-app\backend\app.py:128]
2024-11-07 13:22:39,128 ERROR: Error fetching sheets: type object 'Sheet' has no attribute 'updated_at' [in C:\Users\m\Desktop\web-based excel\custom-excel-app\backend\app.py:128]
2024-11-07 13:22:42,753 ERROR: Error fetching sheets: type object 'Sheet' has no attribute 'name' [in C:\Users\m\Desktop\web-based excel\custom-excel-app\backend\app.py:128]
2024-11-07 13:22:42,794 ERROR: Error fetching sheets: type object 'Sheet' has no attribute 'name' [in C:\Users\m\Desktop\web-based excel\custom-excel-app\backend\app.py:128]
2024-11-07 13:22:42,907 ERROR: Error fetching sheets: type object 'Sheet' has no attribute 'name' [in C:\Users\m\Desktop\web-based excel\custom-excel-app\backend\app.py:128]
2024-11-07 13:22:44,073 ERROR: Error fetching sheets: type object 'Sheet' has no attribute 'name' [in C:\Users\m\Desktop\web-based excel\custom-excel-app\backend\app.py:128]
2024-11-07 13:22:44,208 ERROR: Error fetching sheets: type object 'Sheet' has no attribute 'name' [in C:\Users\m\Desktop\web-based excel\custom-excel-app\backend\app.py:128]
2024-11-07 13:22:44,385 ERROR: Error fetching sheets: type object 'Sheet' has no attribute 'updated_at' [in C:\Users\m\Desktop\web-based excel\custom-excel-app\backend\app.py:128]
2024-11-07 13:22:49,327 ERROR: Error creating sheet: __init__() got an unexpected keyword argument 'name' [in C:\Users\m\Desktop\web-based excel\custom-excel-app\backend\app.py:163]
2024-11-07 13:27:20,712 ERROR: Error fetching sheets: type object 'Sheet' has no attribute 'updated_at' [in C:\Users\m\Desktop\web-based excel\custom-excel-app\backend\app.py:128]
2024-11-07 13:32:06,713 ERROR: Error fetching sheets: type object 'Sheet' has no attribute 'updated_at' [in C:\Users\m\Desktop\web-based excel\custom-excel-app\backend\app.py:128]
2024-11-07 13:32:06,717 ERROR: Error fetching sheets: type object 'Sheet' has no attribute 'updated_at' [in C:\Users\m\Desktop\web-based excel\custom-excel-app\backend\app.py:128]
2024-11-07 13:32:56,576 ERROR: Error fetching sheets: type object 'Sheet' has no attribute 'updated_at' [in C:\Users\m\Desktop\web-based excel\custom-excel-app\backend\app.py:128]
2024-11-07 13:32:56,581 ERROR: Error fetching sheets: type object 'Sheet' has no attribute 'updated_at' [in C:\Users\m\Desktop\web-based excel\custom-excel-app\backend\app.py:128]
2024-11-07 13:33:06,127 ERROR: Error creating sheet: __init__() got an unexpected keyword argument 'name' [in C:\Users\m\Desktop\web-based excel\custom-excel-app\backend\app.py:163]
2024-11-07 13:33:22,440 ERROR: Error fetching sheets: type object 'Sheet' has no attribute 'updated_at' [in C:\Users\m\Desktop\web-based excel\custom-excel-app\backend\app.py:128]
2024-11-07 13:33:22,833 ERROR: Error fetching sheets: type object 'Sheet' has no attribute 'name' [in C:\Users\m\Desktop\web-based excel\custom-excel-app\backend\app.py:128]
2024-11-07 13:33:23,033 ERROR: Error fetching sheets: type object 'Sheet' has no attribute 'name' [in C:\Users\m\Desktop\web-based excel\custom-excel-app\backend\app.py:128]
2024-11-07 13:33:23,316 ERROR: Error fetching sheets: type object 'Sheet' has no attribute 'name' [in C:\Users\m\Desktop\web-based excel\custom-excel-app\backend\app.py:128]
2024-11-07 13:33:23,355 ERROR: Error fetching sheets: type object 'Sheet' has no attribute 'name' [in C:\Users\m\Desktop\web-based excel\custom-excel-app\backend\app.py:128]
2024-11-07 13:33:23,584 ERROR: Error fetching sheets: type object 'Sheet' has no attribute 'name' [in C:\Users\m\Desktop\web-based excel\custom-excel-app\backend\app.py:128]
2024-11-07 13:33:23,821 ERROR: Error fetching sheets: type object 'Sheet' has no attribute 'name' [in C:\Users\m\Desktop\web-based excel\custom-excel-app\backend\app.py:128]
2024-11-07 13:33:24,075 ERROR: Error fetching sheets: type object 'Sheet' has no attribute 'name' [in C:\Users\m\Desktop\web-based excel\custom-excel-app\backend\app.py:128]
2024-11-07 13:33:24,283 ERROR: Error fetching sheets: type object 'Sheet' has no attribute 'name' [in C:\Users\m\Desktop\web-based excel\custom-excel-app\backend\app.py:128]
2024-11-07 13:33:24,455 ERROR: Error fetching sheets: type object 'Sheet' has no attribute 'name' [in C:\Users\m\Desktop\web-based excel\custom-excel-app\backend\app.py:128]
2024-11-07 13:33:24,609 ERROR: Error fetching sheets: type object 'Sheet' has no attribute 'name' [in C:\Users\m\Desktop\web-based excel\custom-excel-app\backend\app.py:128]
2024-11-07 13:33:24,793 ERROR: Error fetching sheets: type object 'Sheet' has no attribute 'name' [in C:\Users\m\Desktop\web-based excel\custom-excel-app\backend\app.py:128]
2024-11-07 13:33:24,939 ERROR: Error fetching sheets: type object 'Sheet' has no attribute 'updated_at' [in C:\Users\m\Desktop\web-based excel\custom-excel-app\backend\app.py:128]
2024-11-07 13:33:25,098 ERROR: Error fetching sheets: type object 'Sheet' has no attribute 'updated_at' [in C:\Users\m\Desktop\web-based excel\custom-excel-app\backend\app.py:128]
2024-11-07 13:33:28,728 ERROR: Error fetching sheets: type object 'Sheet' has no attribute 'updated_at' [in C:\Users\m\Desktop\web-based excel\custom-excel-app\backend\app.py:128]
2024-11-07 13:33:28,740 ERROR: Error fetching sheets: type object 'Sheet' has no attribute 'updated_at' [in C:\Users\m\Desktop\web-based excel\custom-excel-app\backend\app.py:128]
2024-11-07 13:33:37,279 ERROR: Error creating sheet: __init__() got an unexpected keyword argument 'name' [in C:\Users\m\Desktop\web-based excel\custom-excel-app\backend\app.py:163]
2024-11-07 13:35:56,035 ERROR: Error creating sheet: __init__() got an unexpected keyword argument 'name' [in C:\Users\m\Desktop\web-based excel\custom-excel-app\backend\app.py:163]
2024-11-07 13:37:19,484 ERROR: Error fetching sheets: type object 'Sheet' has no attribute 'updated_at' [in C:\Users\m\Desktop\web-based excel\custom-excel-app\backend\app.py:128]
2024-11-07 13:37:19,490 ERROR: Error fetching sheets: type object 'Sheet' has no attribute 'updated_at' [in C:\Users\m\Desktop\web-based excel\custom-excel-app\backend\app.py:128]
2024-11-07 13:37:24,752 ERROR: Error creating sheet: __init__() got an unexpected keyword argument 'name' [in C:\Users\m\Desktop\web-based excel\custom-excel-app\backend\app.py:169]
2024-11-07 13:43:04,684 ERROR: Error fetching sheets: type object 'Sheet' has no attribute 'updated_at' [in C:\Users\m\Desktop\web-based excel\custom-excel-app\backend\app.py:128]
2024-11-07 13:43:31,685 ERROR: Error fetching sheets: type object 'Sheet' has no attribute 'updated_at' [in C:\Users\m\Desktop\web-based excel\custom-excel-app\backend\app.py:128]
2024-11-07 13:43:31,688 ERROR: Error fetching sheets: type object 'Sheet' has no attribute 'updated_at' [in C:\Users\m\Desktop\web-based excel\custom-excel-app\backend\app.py:128]
2024-11-07 13:49:22,535 INFO: Fetching sheets with search term:  [in C:\Users\m\Desktop\web-based excel\custom-excel-app\backend\app.py:123]
2024-11-07 13:49:22,640 INFO: Fetching sheets with search term:  [in C:\Users\m\Desktop\web-based excel\custom-excel-app\backend\app.py:123]
2024-11-07 13:49:22,647 INFO: Fetching sheets with search term:  [in C:\Users\m\Desktop\web-based excel\custom-excel-app\backend\app.py:123]
2024-11-07 13:49:22,655 INFO: Fetching sheets with search term:  [in C:\Users\m\Desktop\web-based excel\custom-excel-app\backend\app.py:123]
2024-11-07 13:49:40,959 INFO: Received sheet creation request: {'name': 'Odoo', 'description': 'this is the very first sheet'} [in C:\Users\m\Desktop\web-based excel\custom-excel-app\backend\app.py:158]
2024-11-07 13:49:40,971 INFO: Sheet created successfully with ID: 192 [in C:\Users\m\Desktop\web-based excel\custom-excel-app\backend\app.py:176]
2024-11-07 13:49:40,977 INFO: Fetching sheets with search term:  [in C:\Users\m\Desktop\web-based excel\custom-excel-app\backend\app.py:123]
2024-11-07 13:49:51,391 INFO: Fetching sheets with search term:  [in C:\Users\m\Desktop\web-based excel\custom-excel-app\backend\app.py:123]
2024-11-07 13:49:51,398 INFO: Fetching sheets with search term:  [in C:\Users\m\Desktop\web-based excel\custom-excel-app\backend\app.py:123]
2024-11-07 13:49:51,407 INFO: Fetching sheets with search term:  [in C:\Users\m\Desktop\web-based excel\custom-excel-app\backend\app.py:123]
2024-11-07 13:49:51,417 INFO: Fetching sheets with search term:  [in C:\Users\m\Desktop\web-based excel\custom-excel-app\backend\app.py:123]
2024-11-07 13:50:01,144 INFO: Fetching sheets with search term:  [in C:\Users\m\Desktop\web-based excel\custom-excel-app\backend\app.py:123]
